compiler/src/iree/compiler/Codegen/Utils/GPUUtils.cpp

```diff
@@ -584,6 +584,8 @@ std::optional<SmallVector<int64_t>> getWmmaNativeVectorSize(Operation *op) {
     return nativeSize;
   }
   if (auto writeOp = dyn_cast<vector::TransferWriteOp>(op)) {
+    if (writeOp.getVectorType().getRank() < 2)
+      return std::nullopt;
```

Comment:
In advance of improvements to wmma/mma.sync unrolling this is necessary to avoid crashes for unrolling ops that shouldn't be unrolled.